PR.jenkinsfile

```diff
@@ -36,5 +36,13 @@ pipeline {
 				sh 'make test'
 			}
 		}
+
+		stage('Test Baseline Scenarios') {
+			steps {
+				catchError(buildResult: 'FAILURE', stageResult: 'FAILURE') {
```

Comment:
We do not need to use `catchError`, right? If I understand documentation corrctly, this construct catches the error and let the pipeline continue. Since we catch the error and mark the result `FAILURE` anyway, we can avoid this constrct at all.

Reply:
`catchError` is removed.